src/app/_hooks/api/useSearch.js

```diff
@@ -14,12 +14,12 @@ export const useListEntries = searchParams =>
 
 export const usePaginatedEntries = searchParams =>
   useInfiniteQuery({
+    // this is needed for auto-invalidate query when changing url query params
     queryKey: [searchParams.toString()],
     queryFn: ({ pageParam = '' }) => {
-      const params = {
-        ...Object.fromEntries(searchParams.entries()),
-        lastCursor: pageParam,
-      };
+      // just in case there is immutable instance passed
```

Comment:
just remove the comment

Reply:
this one was tricky for me, and also i thought it can be non obvious why someone would pass `searchParams` into new `URLSearchParams`, so I've left that intentionally for someone like me :)